cookiecutter/repository.py

```diff
@@ -41,6 +41,20 @@ def expand_abbreviations(template, abbreviations):
     return template
 
 
+def valid_repository(repo_directory):
```

Comment:
I agree. Explicit is better than implicit. However, the function name `valid_repository` isn't correct unless it tests for JSON validity? Perhaps change the name to `repository_has_cookiecutter_json`? Hmmm... that's unweildy...